src/components/panels/transitions.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+	<panel-wrapper :content-class="['panel-transitions']">
+		<template slot="name">Transitions</template>
+		<div class="transitions" v-if="allTransitions">
+			<button
+			  class="transition"
+				v-for="transition in allTransitions"
+				:key="transition.name"
+				:class="[transition.name === currentTransition ? 'is-active' : 'is-inactive']"
+				@click="switchTransitions(transition.name)"
+				v-text="transition.name" />
+			<hr/>
```

Comment:
Maybe separating this into separate transitions list and transition duration panels?

Reply:
can do (npm run lint). (I am a newb to Vue.js and feel like a novice at npm still). Also I made some source code cleanups. will re-submit a new pull.

Reply:
Will look into that. would absolutely be a good idea to relocate the duration list!